pysatModels/tests/test_utils_extract.py

```diff
@@ -357,3 +357,120 @@ def test_success_for_some_already_ran_data(self):
                 assert len(self.inst.data[tcol][
                     ~np.isnan(self.inst.data[tcol])]) > 0
         return
+
+
+class TestUtilsAltitudePressure(object):
```

Comment:
Add a skip statement for pysat version or something similar.  This will allow the tests to pass before the release.